### PR TITLE
Fix lint - errcheck

### PR DESCRIPTION
**Pull request by mattverse**
This PR fixes all lint errors - mainly err check throughout the whole code.

### CODE REVIEW

**Comment by ValarDragon** [comment]
Can you investigate / fix the fialing test?

File: x/claim/keeper/claim.go [hunk 1]
```diff
@@ -25,7 +25,10 @@ func (k Keeper) CreateModuleAccount(ctx sdk.Context, amount sdk.Coin) {
 	moduleAcc := authtypes.NewEmptyModuleAccount(types.ModuleName, authtypes.Minter)
 	k.accountKeeper.SetModuleAccount(ctx, moduleAcc)
 
-	k.bankKeeper.MintCoins(ctx, types.ModuleName, sdk.NewCoins(amount))
+	if err := k.bankKeeper.MintCoins(ctx, types.ModuleName, sdk.NewCoins(amount)); err != nil {
+		return
```
Comment (ValarDragon) [inline x/claim/keeper/claim.go:29]: Lets panic here?

File: x/claim/module.go [hunk 2]
```diff
@@ -73,7 +73,9 @@ func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Rout
 
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the module.
 func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
-	types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
```
Comment (ValarDragon) [inline x/claim/module.go:76]: I'd prefer if we added a // ignore:errcheck, and got that working. Or at least leave `err := types.RegisterQuery...` on its own line, to maintain readability?

File: x/epochs/module.go [hunk 3]
```diff
@@ -83,7 +83,9 @@ func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Rout
 
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the module.
 func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
-	types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
+		return
+	}
```
Comment (ValarDragon) [inline x/epochs/module.go:88]: I'd prefer if we added a // ignore:errcheck, and got that working. Or at least leave err := types.RegisterQuery... on its own line, to maintain readability?

File: x/gamm/module.go [hunk 4]
```diff
@@ -65,7 +65,9 @@ func (b AppModuleBasic) RegisterRESTRoutes(ctx client.Context, r *mux.Router) {
 }
 
 func (b AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
-	types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
+		return
+	}
```
Comment (ValarDragon) [inline x/gamm/module.go:70]: I'd prefer if we added a // ignore:errcheck, and got that working. Or at least leave err := types.RegisterQuery... on its own line, to maintain readability?

File: x/claim/module.go (hunk 2 above)
Comment (ValarDragon) [reply x/claim/module.go:76]: https://golangci-lint.run/usage/false-positives/

We do // nolint:errcheck

**Comment by mattverse** [comment]
Out of curiosity, why do you think err check for these places are unnecessary?

File: x/gamm/module.go (hunk 4 above)
Comment (mattverse) [reply x/gamm/module.go:70]: Out of curiosity, why do you think err check for these places are unnecessary?

**Pull request merged**